monero-outreach-docs/translations/nl/PSA/files/teamwork.md

```diff
@@ -0,0 +1,13 @@
+# Deelname binnen de Monero gemeenschap
```

Comment:
# Gemeenschapsdeelname

Monero is een volledig gedecentraliseerd, open-bronproject. Iedereen met een good idee en wat initiatief kan een Monero-project creëren en leiden. Ben je geen programmeur maar anderzijds geïnteresseerd in Monero, dan zijn er toch nog meerdere opties. Monero-projecten organiseren zich in verschillende werkgroepen. Er is de Lokaliseringswerkgroep die alle inhoud vertaalt om Monero globaal te verspreiden, de Uitreikwerkgroep die tools creëert voor betere publieke relaties en adoptie, en zelfs een werkgroep voor een Monero hardware wallet. Ook vele andere werkgroepen hebben je hulp nodig.

Ben je een programmeur, dan kan je je bij een paar honderd andere programmeurs voegen die bijgedragen hebben aan de code, via GitHub.

De eerste halte wat betreft deelname aan een gemeenschapswerkgroep is de Monero-gemeenschapswerkgroep, die alle andere werkgroepen organiseert. Zij kunnen je zeggen of je idee al dan niet nieuw is, en met welke mensen je kan werken en een stapje verder kan gaan. Of als je idee totaal nieuw is, kunnen ze hulpmiddelen aanbieden om je idee te helpen realiseren.

Monero's kernprogrammeurs hebben een handvol tools verschaft voor het succes van werkgroepen, onder meer Mattermost voor communicatie, en Taiga voor projectplanning en task flow management. Personen kunnen ook deelnemen in de levendige discussies op de verschillende kanalen in Slack en IRC, via de vele verschillende Telegram group chats, op Reddit via r/Monero, of op de vele subreddits toegeweid tot niche-onderwerpen omtrent Monero (mijnen, speculatie,..), en door vragen over Monero te stellen of te beantwoorden op Stack Exchange.

### De enige hindernis tot deelname in de gemeenschap is jezelf

We nodigen je uit je tot onze rangen te voegen en deze gedecentraliseerde open-brongemeenschap nog sterker te maken!